src/webots/core/WbNetwork.cpp

```diff
@@ -118,28 +114,28 @@ void WbNetwork::save(const QString &url, const QByteArray &content) {
       mCacheSizeInBytes += file.size();
       file.close();
       // save reference in internal representation
-      gCacheMap.insert(url, path);
+      mCacheMap.insert(url, path);
     }
   }
 }
 
-const QString &WbNetwork::get(const QString &url) {
-  if (!gCacheMap.contains(url)) {
+const QString WbNetwork::get(const QString &url) {
```

Comment:
Why did you remove the reference here?

Reply:
This reference was not present before the changes applied by @stefaniapedrazzi in #5023. So I thought it was not useful. If we use a reference for the `get` method, should we also use one for `getUrlFromEphemeralCache`?

Reply:
Yes, that's probably better.

Reply:
Actually, `getUrlFromEphemeralCache` is returning a temporary object, so it is not a good idea to use a reference. I still put it back for the `get` method.

Reply:
We should always return the reference instead of a copy when possible.